### PR TITLE
Update UI Theme

### PR DESCRIPTION
**Pull request by Noxsios**
Updates theme to utilize newly created `Theme` component - https://unicorn-ui.netlify.app/theme

- [x] Updates theme (migration from `app.css` --> `lib/palette.ts`)
- [x] Adds functionality for theme toggle in upper right corner
- [x] Changes default theme based on device settings

### CODE REVIEW

**Comment by Noxsios** [comment]
PR is ready to review as is, then can be re-reviewed when typography theming is added to Unicorn UI.

**Comment by jeff-mccoy** [comment]
screenshots or videos plz

**Comment by Noxsios** [comment]
> screenshots or videos plz


https://user-images.githubusercontent.com/50058333/211412081-d600c5b4-aafb-4867-8110-4e4ef7edcdf8.mov

File: src/ui/lib/components/header.svelte [hunk 1]
```diff
@@ -37,7 +38,8 @@
 		display: flex;
 		align-items: center;
 		background-color: #0d133d;
-		color: #ffffffdd;
+		// palette.shared.text.secondary.onDark
+		color: rgba(255, 255, 255, 0.7);
```
Comment (mike-winberry) [inline src/ui/lib/components/header.svelte:42]: I believe this can just be var(--mdc-theme-text-secondary-on-dark)

**Review by mike-winberry — CHANGES_REQUESTED** [review]

File: src/ui/lib/components/header.svelte (hunk 1 above)
Comment (Noxsios) [reply src/ui/lib/components/header.svelte:42]: you right, would be good to add the the docs on theme usage on how the palette object can be translated to css vars

**Review by mike-winberry — APPROVED** [review]

**Pull request merged**